.github/workflows/test.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+
+permissions:
+  contents: read
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  intel_build:
+    name: Intel OneAPI build
+    runs-on: ubuntu-20.04
+    env:
+      FC: mpiifort
+      APT_PACKAGES: >-
+        intel-oneapi-compiler-fortran
+        intel-oneapi-mpi
+        intel-oneapi-mpi-devel
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 2
+
+    - name: Add Intel repository
+      run: |
+        wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
+        sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
+        rm GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
+        echo "deb https://apt.repos.intel.com/oneapi all main" | sudo tee /etc/apt/sources.list.d/oneAPI.list
+        sudo apt-get update
```

Comment:
FFTW probably needs to be compiled with the same compiler as qdyn, so you need to move the FFTW compilation after you setup the intel compiler. 

I'd recommend just copying the `fftw_build` job from ABIN, where I install fftw via apt to avoid compilation. In that case, you need to use to default gfortran compiler otherwise the installed fftw apt package will not be compatible.

Reply:
> Wow, nothing escapes your attention on GitHub.

Haha, for some reason I am getting emails for each change on the qdyn repo :-D I was getting a lot of them today. :-P

Reply:
Ok, cool, thanks for the advice, I will try it someday. 

> Haha, for some reason I am getting emails for each change on the qdyn repo :-D I was getting a lot of them today. :-P

I had no idea you were watching this repo. I was just playing around thinking no one would see it. :D